sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -48,7 +51,16 @@ public synchronized Long getAppStartInterval() {
     if (appStartMillis == null || appStartEndMillis == null || coldStart == null) {
       return null;
     }
-    return appStartEndMillis - appStartMillis;
+    final long appStart = appStartEndMillis - appStartMillis;
+
+    // we filter out app start more than 60s.
+    // this could be due to many different reasons.
```

Comment:
I assume we know some of the reasons. Could we add them to make it easier for people coming across this comment to understand why it's necessary?

Reply:
Makes sense, thanks.